### PR TITLE
fix: Support HTML plugin

### LINKED ISSUE #3: In the case of HTML Paragraph, erroneous detection occurs.
I applied the no-start-duplicated-conjunction rule to the HTML file.
The following Lint error occurred, but I think that this error is a false detection.

```console
  916:1011  error    Don't repeat "</p>" in 2 phrases    no-start-duplicated-conjunction
```

In the case of HTML, the data of the node of Paragraph is as follows.

```console
{ type: 'Paragraph',
  tagName: 'p',
  properties: {},
  children: 
   [ { type: 'Strong',
       tagName: 'strong',
       properties: {},
       children: [Array],
       loc: [Object],
       range: [Array],
       raw: '<strong>HTML と、Page クラスに Footer 部とする Container を追加する</strong>' },
     type: 'UNKNOWN' ],
  loc: 
   { start: { line: 1144, column: 36 },
     end: { line: 1144, column: 104 } },
  range: [ 46454, 46522 ],
  raw: '<p><strong>HTML と、Page クラスに Footer 部とする Container を追加する</strong></p>' }
```

In the case of Markdown, the data of the node is as follows.

```console
{ type: 'Paragraph',
  children: 
   [ { type: 'Code',
       value: 'o',
       loc: [Object],
       range: [Array],
       raw: '`o`' },
     { type: 'Str',
       value: ' をつけたルールを一律 グローバルインストールします。    ',
       loc: [Object],
       range: [Array],
       raw: ' をつけたルールを一律 グローバルインストールします。    ' } ],
  loc: 
   { start: { line: 117, column: 0 },
     end: { line: 117, column: 34 } },
  range: [ 5761, 5795 ],
  raw: '`o` をつけたルールを一律 グローバルインストールします。    ' }
```

I thought false detection occurred because HTML tags included in the `raw` property of the Paragraph node generated from HTML.
I attempted to fix it. but I could not decide whether to remove HTML tags on the rule side or whether HTML parser should remove the HTML tag of `raw` property.

### PR DESCRIPTION
**Pull request by kemsakurai**
Support HTML plugin
* Use textlint-util-to-string to get `raw` string.
* Adjusted the position of index using the originalIndexFromIndex
method.

----
Edit by @azu

fix #3

### CODE REVIEW

File: src/no-start-duplicated-conjunction.js [hunk 1]
```diff
@@ -55,7 +57,11 @@ module.exports = function(context, options = {}) {
                 }
                 if (useDuplicatedPhase) {
                     const sentenceStartIndex = node.range[0] + sentence.range[0];
-                    if (!ignoreNodeManager.isIgnoredIndex(sentenceStartIndex)) {
+                    const originalIndex = source.originalIndexFromIndex(sentenceStartIndex);
+                    // adjust index 
+                    // if  "また、[import, a.js](a.js)" then originalIndex is used.
+                    // if "[import, binary-example.js](src/binary-example.js)" then  originalIndex is undefined.
+                    if (!ignoreNodeManager.isIgnoredIndex(typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex)) {
```
Comment (azu) [inline src/no-start-duplicated-conjunction.js]: > typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex

It is better that assign to variable like following:

```js
const targetIndex = typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex
```

**Review by azu — APPROVED** [review]
I've commented.
Other is LGTM

File: src/no-start-duplicated-conjunction.js (hunk 1 above)
Comment (kemsakurai) [reply src/no-start-duplicated-conjunction.js]: Thanks for the review.
I think I will fix it.

**Comment by azu** [comment]
Release https://github.com/azu/textlint-rule-no-start-duplicated-conjunction/releases/tag/1.1.4

**Pull request merged**